metadata-ingestion/src/datahub/ingestion/source/bigquery_v2/bigquery.py

```diff
@@ -87,8 +88,11 @@
     TagAssociationClass,
 )
 from datahub.utilities.mapping import Constants
+from datahub.utilities.perf_timer import PerfTimer
 from datahub.utilities.registries.domain_registry import DomainRegistry
 
+TABLE_PER_BATCH = 50
```

Comment:
Should this be a config param instead? If not, make it local to `get_tables_for_dataset`.